nss-evening-cohort-24/pos-system-calzoned-in
# Orderform setup

## Description
Created the addOrderForm and the click event for the navBar to pull the addOrderForm up.   
Made the addOrderForm have a dropdown to select the Order Type, as well as being required to select before submitting the form.

## Related Issue





## Motivation and Context
Originally I had checkboxes for the Order Type but they were not required so I changed it to a dropdown that is blank so you get an alert if you try to submit it blank.
This will require every order to have an ordertype.

## How Can This Be Tested?
I added a console.warn to the click event to make sure it showed up in the console.  
Also filled out the form many times to ensure that all fields had to be filled out before being submitted.

## Screenshots (if appropriate):
![image](https://github.com/nss-evening-cohort-24/pos-system-calzoned-in/assets/124642873/cd270d80-441d-4a9e-9277-de2fbce12b12)

## Types of changes

- [x ] New feature (non-breaking change which adds functionality)
